firmware/libraries/versavis/src/ADIS16448AMLZ.cpp

```diff
@@ -386,20 +386,32 @@ float ADIS16448AMLZ::magnetometerScale(int16_t sensorData) {
 // Method to update the sensor data without any validity checks (may result in
 // spikes).
 ///////////////////////////////////////////////////////////////////////////////////////////////
-void ADIS16448AMLZ::updateData() { sensor_data_ = sensorReadWoCRC(); }
+bool ADIS16448AMLZ::updateData() {
+  sensor_data_ = sensorReadWoCRC();
+  return true;
```

Comment:
`return sensor_data_ != nullptr`? would be better than just `return true`. Same for all the other `updateData()`